src/firefly/js/ui/UploadTableSelector.jsx

```diff
@@ -0,0 +1,172 @@
+import PropTypes from 'prop-types';
+import {FieldGroupCtx} from 'firefly/ui/FieldGroup';
```

Comment:
remember this file will have to be ported to JoyUI based on what was already done with `UploadTableSelector`

Reply:
got it. this will be the main work then I suppose in the rebase after I merge this PR?